### PR TITLE
New branch and copied file RepeatedPlanning.adoc as unable to resolve…

### PR DESCRIPTION
**Pull request by gsheldon**
… conflicts on previous pull request

### CODE REVIEW

File: optaplanner-docs/src/main/asciidoc/RepeatedPlanning/RepeatedPlanning-chapter.adoc [hunk 1]
```diff
@@ -56,54 +86,68 @@ There are two ways to avoid that:
 
 If we handle overconstrained planning with nullable variables, the overload entities will be left unassigned:
 
-image::RepeatedPlanning/overconstrainedPlanning.png[align="center"]
+image::RepeatedPlanning/OverconstrainedPlanning/overconstrainedPlanning.png[align="center"]
```
Comment (ge0ffrey) [inline optaplanner-docs/src/main/asciidoc/RepeatedPlanning/RepeatedPlanning-chapter.adoc]: Doesn't this break the image?
This PR only changes 1 file and that png is not in that new directory.

File: optaplanner-docs/src/main/asciidoc/RepeatedPlanning/RepeatedPlanning-chapter.adoc [hunk 2]
```diff
@@ -56,54 +86,68 @@ There are two ways to avoid that:
...
-. Optionally change all soft constraints to ignore virtual assigned entities.
-
+** Do not add too many, as that will decrease solver efficiency.
+** Importantly, do not add too few as that will lead to an infeasible solution.
+. Add a score constraint on the new level (usually a medium constraint) to penalize the number of virtual assigned entities (or a weighted sum of them).
+. Optionally, change all soft constraints to ignore virtual assigned entities.
 
 [[continuousPlanning]]
-== Continuous Planning (Windowed Planning)
+=== Continuous Planning (Windowed Planning)
```
Comment (ge0ffrey) [inline optaplanner-docs/src/main/asciidoc/RepeatedPlanning/RepeatedPlanning-chapter.adoc]: Continuous Planning should not be a subsection of Overconstrained Planning.

**Comment by ge0ffrey** [comment]
Besides the 2 comments (which affect some other lines too), this looks good.
I wasn't able to validate if no content is disappearing, but I thrust that's fine.

Let me know when the 2 comments are fixed and I 'll merge.

**Comment by gsheldon** [comment]
Hoping this is all resolved and ok to merge when ready.

**Comment by ge0ffrey** [comment]
@gsheldon Thanks Gemma :)

**Comment by ge0ffrey** [comment]
To avoid the bunch of commits, I squashed them (to do this locally, use "git rebase -i").
The diff was too big for me to confirm than no info was lost, but because it was only file that changed, I checked if recent changes to that file weren't undone and they weren't.

**Pull request merged**